modules/blocks/utils.js

```diff
@@ -322,8 +323,11 @@ Utils.prototype.loadBlocksData = function(filter, cb, tx) {
 	}
 
 	// Get height of block with supplied ID
-	return (tx || library.db).blocks
-		.getHeightByLastId(filter.lastId || null)
+	return library.storage.entities.Block.get(
+		{ id: filter.lastId || null },
+		{ limit: params.limit },
```

Comment:
Actually sorting and limiting here are unnecessary because if `lastId` is present it will return only one row and if `lastId` is not present it will return no rows.

Reply:
We decided to refactor this method in seperate issue.